app/routers/dayview.py

```diff
@@ -105,6 +105,20 @@ def _check_multiday_event(self) -> Tuple[bool]:
 
 
 def event_in_day(event: Event, day: datetime, day_end: datetime) -> bool:
+    if event.all_day:
+        return None
+    return (
+        (event.start >= day and event.start < day_end) or
+        (event.end >= day and event.end < day_end) or
+        (event.start < day_end < event.end)
+        )
```

Comment:
Isn't this couldn't be rephrased as `not all_day_event`?

Reply:
If I understand it correctly it checks whether the event occurs on a certain day or not- thus, can not be replaced...
I didn't write this function just added the conditioning:)